arviz/plots/backends/bokeh/forestplot.py

```diff
@@ -270,78 +274,116 @@ def display_multiple_ropes(self, rope, ax, y, linewidth, rope_var):
         )
         return ax
 
-    def ridgeplot(self, mult, linewidth, alpha, ridgeplot_kind, ridgeplot_quantiles, ax):
+    def ridgeplot(
+        self,
+        hdi_prob,
+        mult,
+        linewidth,
+        markersize,
+        alpha,
+        ridgeplot_kind,
+        ridgeplot_truncate,
+        ridgeplot_quantiles,
+        ax,
+    ):
         """Draw ridgeplot for each plotter.
 
         Parameters
         ----------
+        hdi_prob : float
+            Probability for the highest density interval. Defaults to 0.94
```

Comment:
```suggestion
            Probability for the highest density interval.
```

I would remove it as this particular function has no default, otherwise I'd say it defaults to value in rcParams

Reply:
Similar comments below, not sure about documenting the defaults here at the lower level where they have already been enforced and hopefully documented, I think this could easily end up being outdated due to changes in the general function that do not modify the docs here.